src/test/java/tutorlink/command/FindStudentCommandTest.java

```diff
@@ -0,0 +1,80 @@
+//@@author RCPilot1604
+package tutorlink.command;
+
+import java.util.HashMap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import tutorlink.appstate.AppState;
+import tutorlink.exceptions.IllegalValueException;
+import tutorlink.exceptions.StudentNotFoundException;
+import tutorlink.result.CommandResult;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class FindStudentCommandTest {
+    private FindStudentCommand findCommand;
+    private AddStudentCommand addCommand;
+    private final AppState appState = new AppState();
+    private HashMap<String, String> arguments;
+    private CommandResult result;
```

Comment:
Class-level variables result and arguments should be local to the methods that use them. This reduces the risk of state-related bugs in tests.